modulefiles/ufs_acorn.intel

```diff
@@ -0,0 +1,26 @@
+#%Module
+
+proc ModulesHelp {} {
+  puts stderr "\tcit - loads modules required for building and running UFS Model on Acorn"
+}
+
+module-whatis "loads UFS Model prerequisites for Acorn"
+
+module load PrgEnv-intel/8.1.0
+module load intel/19.1.3.304
+module load craype/2.7.13
+module load cray-mpich/8.1.7
+
+module load cmake/3.20.2
+
+module use /lfs/h1/emc/nceplibs/noscrub/hpc-stack/libs/hpc-stack/modulefiles/stack
+module load hpc/1.2.0
```

Comment:
It looks to me we are using hpc/1.1.0 on other platforms (hera,orion etc). Is there any difference between hpc/1.2.0 and hpc/1.1.0?